cmd/tailscale/cli/risks.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cli
+
+import (
+	"flag"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+)
+
+const (
+	riskLoseSSH = "lose-ssh"
+)
+
+var (
+	riskMessages = map[string]string{
+		riskLoseSSH: `You are connected over Tailscale, this action may result in your session disconnecting.`,
+	}
+	acceptedRisks string
+)
+
+func riskTypes() (r []string) {
+	r = make([]string, 0, len(riskMessages))
+	for k := range riskMessages {
+		r = append(r, k)
+	}
+	sort.Strings(r)
+	return r
+}
+
+// registerAcceptRiskFlag registers the --accept-risk flag. Accepted risks are accounted for
+// in presentRiskToUser.
+func registerAcceptRiskFlag(f *flag.FlagSet) {
+	f.StringVar(&acceptedRisks, "accept-risk", "", "accept risk and skip confirmation for risk types: "+strings.Join(riskTypes(), ","))
+}
+
+// riskAccepted reports whether riskType is in acceptedRisks.
+func riskAccepted(riskType string) bool {
```

Comment:
This func isn't exactly correct in general.

You could just write the allocating straightforward version too.